### ISSUE TITLE
Question about latest version of todoist for windows

### ISSUE THREAD

**Issue by dylan-k** [rickstaa/todoist-global-shortcuts-WIN10#8]
It seems the new version of Todoist for Windows is basically a browser for the todoist site and also lacks the global shortcut for "add task." Would this workaround be compatible with the new version as well?

**Comment by rickstaa** [comment]
@dylan-k Thanks a lot for your interest in my workaround. Sadly the workaround does not seem to work with the new Todoist app. I created the new AHK key that works with the new app. I will release it under version 2.0 later this week. You can already replace the code in the `Todoist_global_shortcuts.ahk` script with the code below to get it to work.

````ahk
#NoEnv  ; Recommended for performance and compatibility with future AutoHotkey releases.
; #Warn  ; Enable warnings to assist with detecting common errors.
SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.

;!DO NOT CHANGE THE CODE AND SETTINGS BELOW UNLESS YOU KNOW WHAT YOU ARE DOING!
SetTitleMatchMode, 2 ; IfWinExist settings
DetectHiddenWindows, On

;--SCRIPT SETTINGS--
WaitTime := 1100 ; Adjust this time if the script is not working on program startup

;--Shortkey code--

; Open Todoist with alt+ctrl+t shortcut
!^t::

Process, Exist, Todoist.exe
IfWinNotExist, ahk_exe Todoist.exe
{
    Run, shell:AppsFolder\com.todoist
    return
}
else
{
    IfWinActive, %Title%
        WinMinimize, %Title%
    else
        Run, shell:AppsFolder\com.todoist
}
return

; Open todoist and add task shortcut
!^a::

Process, Exist, Todoist.exe
IfWinNotExist, ahk_exe Todoist.exe
{
        RunWait, shell:AppsFolder\com.todoist
		sleep, WaitTime ; Waits for Todoist to load
	    WinActivate, %Title%
		{
			send, {q}
			return
		}
}
else
{
    IfWinActive, %Title%
        WinMinimize, %Title%
    else
        Run, shell:AppsFolder\com.todoist
		sleep, WaitTime ; Waits for Todoist to load
	    WinActivate, %Title%
		{
			send, {q}
			return
		}
}
return
````

**Comment by rickstaa** [comment]
@dylan-k I just released the new updated version as [v2.0.0](https://github.com/rickstaa/todoist-global-shortcuts-WIN10/releases/tag/v2.0.0). I tested it on my pc, and it works as expected. Let me know if you have any other problems.
